c10m/c10m_bespoke.c

```diff
@@ -50,7 +50,7 @@ uint32_t resume_async_worker(uint32_t key, uint32_t value) {
 
 extern
 __attribute__((import_module("env"), import_name("free_async_worker")))
-void free_async_worker(uint32_t _unused) {}
+void free_async_worker(__attribute__ ((unused)) uint32_t _unused) {}
```

Comment:
Does the following silence the warning?
```suggestion
void free_async_worker(uint32_t) {}
```

Reply:
Only if we did `--std=c23`. Before that you can only omit parameter names in function declarations, but not definitions.